searchapp/static/student_view.js

```diff
@@ -162,6 +162,51 @@ function upload_click(e) {
 	});
 }
 }
+
+function populate_grades(value,text, $selectedItem) {
+	let worksheetType = document.getElementById("worksheetType") ? $("#worksheetType").dropdown('get value') : 'paper';
+	let boardData = {
+		"board": $(`#${worksheetType}-board`).dropdown('get value')
+	};
+	$.ajax({
+		url: BASE_DIR + "/get_grades",
+		method : "get",
+		data: boardData,
+		headers: { "X-CSRFToken": csrftoken, crossOrigin: false},
+		success: function(response) {
+			$(`#${worksheetType}-grade-options-parent`).empty();
+			$(`#${worksheetType}-grade`).removeClass('hide-display').addClass('show-display');
+			let grades = response['grade_list'];
+			for (let i=0; i<grades.length; i++) {
```

Comment:
Lets start moving away from conventional for loops. Try using `forEach` with arrow functions. Should improve readability;